packages/@vuepress/theme-default/layouts/404.vue

```diff
@@ -3,23 +3,42 @@
     <div class="theme-default-content">
       <h1>404</h1>
       <blockquote>{{ getMsg() }}</blockquote>
-      <router-link to="/">Take me home.</router-link>
+      <router-link :to="localePath">{{ notFoundLinkText }}</router-link>
     </div>
   </div>
 </template>
 
 <script>
-const msgs = [
-  `There's nothing here.`,
-  `How did we get here?`,
-  `That's a Four-Oh-Four.`,
-  `Looks like we've got some broken links.`
-]
-
 export default {
+  computed: {
+    localePath () {
+      return this.$localePath || '/'
+    },
+    notFoundLinkText () {
+      return (
+        this.$themeLocaleConfig.notFoundLinkText
+        || this.$site.themeConfig.notFoundLinkText
+        || `Take me home.`
+      )
+    },
+    notFoundMessages () {
+      const messages = [
```

Comment:
I know that back quotes were already used before, but here you can use simple quotes 😉